WordPress/Classes/ViewRelated/Notifications/Controllers/NotificationsViewController/NotificationsViewController.swift

```diff
@@ -331,8 +332,43 @@ class NotificationsViewController: UIViewController, UIViewControllerRestoration
     }
 
     func tableView(_ tableView: UITableView, cellForRowAt indexPath: IndexPath) -> UITableViewCell {
-        let cell = tableView.dequeueReusableCell(withIdentifier: ListTableViewCell.defaultReuseID, for: indexPath)
-        configureCell(cell, at: indexPath)
+        guard let cell = tableView.dequeueReusableCell(withIdentifier: NotificationsTableViewCellContent.reuseIdentifier) as? HostingTableViewCell<NotificationsTableViewCellContent>,
+              let note = tableViewHandler.resultsController?.managedObject(atUnsafe: indexPath) as? Notification else {
+            return UITableViewCell()
+        }
+        cell.selectionStyle = .none
+        if let deletionRequest = notificationDeletionRequests[note.objectID] {
```

Comment:
[Nit] This if-else is a little bit hard to read, also, the `cell.host` method call is redundant. Why not do this if-else logic only on the `style`?

```swift
let style = {
   if let deletionRequest {
       // return altered style
   }
   // return regular style
}()

cell.host(...)
```